test/binCases/errors/info-verbosity/info-verbosity.test.js

```diff
@@ -17,6 +17,7 @@ test("info-verbosity", () => {
 	]);
 
 	expect(stderr).toContain("Invalid values:");
-	expect(stderr).toContain("Argument: info-verbosity, Given: \"false\", Choices: \"none\", \"info\", \"verbose\"");
+	// eslint-disable-next-line quotes
```

Comment:
here

Reply:
they're not conflicting, remove the lint disable, and use single quotes to escape, I think that might be a solution

Reply:
tried, not working, prettier fails after that :(

Reply:
Lemme see the ci failing on switching a single one

Reply:
Sure, thanks! 🌞